api/src/main/java/org/openmrs/module/aijar/AijarActivator.java

```diff
@@ -155,7 +159,18 @@ public void started() {
             // generate OpenMRS ID for patients without the identifier
             generateOpenMRSIdentifierForPatientsWithout();
 
-
+            // set alert if National Health Provider Identifier is not set
+            if (administrationService.getGlobalProperty(GP_NHPI).equalsIgnoreCase(GP_NHPI_VALUE)) {
```

Comment:
@slubwama Check for a null pointer in case the Global Property does not exist

Reply:
@slubwama Another thought - instead of repeating the checks 3 times can we have a method which takes a Global property and message code which does the checks

Reply:
@slubwama Can you move this code for the messages into an initializer - You can rename the current AppConfigurationInitializer to TaskSetupInitializer and use that class name